rmgpy/rmg/reactors.py

```diff
@@ -308,6 +309,10 @@ def __init__(self, core_phase_system, edge_phase_system, initial_conditions, ter
 
         self.terminations = [to_rms(term) for term in terminations]
 
+    def get_const_spc_indices(self,spcs=None):
+        rms_species_names = [spc.name for spc in self.core_phase_system.get_rms_species_list()]
+        return [rms_species_names.index(name) for name in self.const_spc_names]
```

Comment:
Is this function only used once? If not we should consider doing this once and save the indices

Reply:
I checked and this function actually is unnecessary as the actually operation takes place inside RMS (I think I filled it in because the old reaction_system object needed it). I've told it to ignore that for Reactor objects now.